examples/widgets/bottom_sheet.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class BottomSheetDemo extends StatelessComponent {
+  final TextStyle textStyle = new TextStyle(
+    color: Colors.indigo[400],
+    fontSize: 24.0,
+    textAlign: TextAlign.center
+  );
+
+  void _showModalBottomSheet(BuildContext context) {
+    showModalBottomSheet(context: context, child:
+      new GestureDetector(
+        onTap: () { Navigator.of(context).pop(); },
+        child: new Container(
+          decoration: new BoxDecoration(backgroundColor: Theme.of(context).canvasColor),
+          child: new Padding(
+            padding: const EdgeDims.all(32.0),
+            child: new Text("This is the modal bottom sheet. Click anywhere to dismiss.", style: textStyle)
+          )
+        )
+      )
+    );
+  }
+
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      toolBar: new ToolBar(center: new Text('BottomSheet Demo')),
+      floatingActionButton: new FloatingActionButton(
+        child: new Icon(type: 'content/add', size: 24),
+        backgroundColor: Colors.redAccent[200]
+      ),
+      body: new GestureDetector(
+        onTap: () { _showModalBottomSheet(context); },
```

Comment:
Sure. I was just verifying that while the modal bottom sheet was up, the button couldn't be pressed.